app/controllers/admins/employees_controller.rb

```diff
@@ -38,6 +62,10 @@ def employee_params
       params.require(:employee).permit(:email, :password, :password_confirmation, :number_of_available_kudos)
     end
 
+    def permit_add_kudos
```

Comment:
I would suggest changing the name to something more self-explanatory
```suggestion
    def add_kudos_params
```